build.gradle

```diff
@@ -127,11 +128,45 @@ jacocoTestReport {
 
 project.tasks['sonarqube'].dependsOn jacocoTestReport
 
+pitest {
+  pitestVersion.set('1.5.1')
+  junit5PluginVersion.set('0.12')
+  testSourceSets = [sourceSets.test]
+  mainSourceSets = [sourceSets.main]
+  targetClasses = ['uk.gov.hmcts.reform.bulkscan.orchestrator.*']
+  excludedClasses = [
+    'uk.gov.hmcts.reform.bulkscan.orchestrator.config.*'
+  ]
+  threads = 10
+  enableDefaultIncrementalAnalysis = true
+  outputFormats = ['XML', 'HTML']
+  timestampedReports = false
+  mutationThreshold = 50
+  timeoutConstInMillis = 30000
+  mutators.set([
+    'CONDITIONALS_BOUNDARY',
+    'INCREMENTS',
+    'INVERT_NEGS',
+    'MATH',
+    'NEGATE_CONDITIONALS',
+    'VOID_METHOD_CALLS',
+    'EMPTY_RETURNS',
+    'FALSE_RETURNS',
+    'NULL_RETURNS',
+    'PRIMITIVE_RETURNS',
+    'TRUE_RETURNS',
+    'REMOVE_CONDITIONALS'
```

Comment:
are those set on some kind of programme wide guidelines?

Reply:
if so, can you add a link to it?
Just wondering why those specific mutator are being used here and not the default set for example

Reply:
I used Pcq-backend  and Pit test Website as a reference,The reason for specifying  other than default mutators is that they will apply more mutations on the unit tests along with default mutations.May i know is there any harm by those mutations?